pymc3/model.py

```diff
@@ -734,7 +736,9 @@ def logpt(self):
 
     @property
     def logp_nojact(self):
-        """Theano scalar of log-probability of the model"""
+        """Theano scalar of log-probability of the model but without the jacobian 
+        if transformed Random Variable is presented.
```

Comment:
Does this mean that the Jacobian will be present if there are no transformed random variables? That is how I understand the sentence.

Reply:
If there are no transformed variables there is no need for Jacobian correction, hence logp_nojact will be the same as logpt
This should be rephrased to make it clearer indeed.

Reply:
```suggestion
        if transformed Random Variable is presented.
        Note that If there is no transformed variable in the model, logp_nojact 
        will be the same as logpt as there is no need for Jacobian correction.
```